AliveSFU/AliveSFU.xcodeproj/project.pbxproj

```diff
@@ -889,6 +889,7 @@
 				C699DB9B1DCDD7E50088CAEE /* Release */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
```

Comment:
qwhat I don't remember changing this, I'm assuming this flag is supposed to be set to something like debug rather than release?